e2e_tests/tests/cluster/test_logging.py

```diff
@@ -143,6 +145,37 @@ def do_check_logs() -> None:
             bindings.post_KillShell(sess, shellId=task_id)
 
 
+@pytest.mark.e2e_cpu
+def test_tcd_startup_hook_task_combined() -> None:
+    for ntsc in ["command", "notebook", "shell"]:
+        _test_task_logs(ntsc, {}, re.compile("^.*hello from rp tcd startup hook.*$"))
+
+
+@pytest.mark.e2e_cpu_elastic
+def test_tcd_startup_hook_task_master() -> None:
+    for ntsc in ["command", "notebook", "shell"]:
+        _test_task_logs(ntsc, {}, re.compile("^.*hello from master tcd startup hook.*$"))
+
+
+@pytest.mark.e2e_cpu
+@pytest.mark.e2e_cpu_elastic
+@pytest.mark.e2e_cpu_cross_version
+@pytest.mark.e2e_gpu  # Note, e2e_gpu and not gpu_required hits k8s cpu tests.
+@pytest.mark.e2e_slurm
```

Comment:
I don't know if you have already but would check this passes in slurm

Reply:
I manually tested the older version and I'm checking the new one. any good way of adding this to be run on slurm from this repo? would need a separate follow up pr on ee after we land this or setup a different ee pr and make modifications there.